config/config.default.php

```diff
@@ -275,14 +277,11 @@
     // Enables the planned arrival/leave date
     'enable_planned_arrival'  => (bool) env('ENABLE_PLANNED_ARRIVAL', true),
 
-    // Enables the T-Shirt configuration on signup and profile
-    'enable_tshirt_size'      => (bool) env('ENABLE_TSHIRT_SIZE', true),
-
-    // When true changes everything from shirts to goodies and disables shirt size
-    'other_goodie'            => (bool) env('OTHER_GOODIE', false),
-
-    // Enables the goody configuration on signup and profile
-    'enable_goody'            => (bool) env('ENABLE_GOODY', false),
+    // Resembles the Goodie Type. There are three options:
+    // 'none' => no goodie at all
+    // 'goodie' => a goodie which has no sizing options
+    // 'tshirt' => goodie that is called tshirt and has sizing options
+    'goodie_type' => GoodieType::from(env('GOODIE_TYPE', 'goodie')),
```

Comment:
The only allowed function in config files is env() as it just reads values and static castst. Everything else will confuse users.

The `GoodieType::from` should be used in the code itself instead.

Reply:
where else?

Reply:
done